docs/routing/special_routes.md

```diff
@@ -0,0 +1,56 @@
+# Error handling in router
+
+Сurrently ILC can only handle [404](#404-error) and [5XX](#5xx-error) errors.
+
+Our web page now is composed from several fragments, that's why we can't use the same approach we used with monolithic frontend.
+This document describes how these errors were realized in ILC and why in this way.
+
+## '404' Error
+
+To see 404 error routes, use the `Show special` switcher at the top of the routes page. `Special route` must be created separately for each domain.
+
+More about [404 Error here](../global_errors_handling.md#404-error-not-found).
+
+![ILC registry special routes switcher](../assets/routes/special-routes-switcher.png)
+
+Unlike simple routes which use composition with the help of `Next: true|false` property, the special route has own full configuration. This means that you must add template and all the necessary applications (header, footer, etc.) to the `Special route` [slot](./route_configuration_options.md#slot-configuration) (as in the screenshot below).
+
+![ILC registry error slot](../assets/routes/error-slot.png)
+
+### Example:
+
+Let's go to the `/wrapper/blablabla/` route, ILC requires the route chain to end with a route with the `Next` field parameter: `false`, in our case we don’t have a suitable route at all, which means that ILC will go through the entire list of routes and not finding the closing route (with the `Next: false` parameter) and so it will render a [404 Error](../global_errors_handling.md#404-error-not-found).
+
+![ILC registry route to error](../assets/routes/route.png)
+
+**Result:**
+
+![ILC registry 404 error example](../assets/routes/fourth-case-result.png)
+
+## '5XX' Error
+
+This error is a simple HTML template. It is designed in such a way to not use ILC routing unlike 404 errors, which means that even if something went wrong with ILC, this error will be rendered.
```

Comment:
1. Error couldn't be called a "template". Pls rephrase.
2. It wasn't designed to not to use ILC routing. It's was designed to be a static markup in order to minimise chances of error occurrence during it's rendering. As it's a last chance to show something meaningful to visitor.